content/en/blog/_posts/2024-07-xx-kube-scheduler-simulator/index.md

```diff
@@ -0,0 +1,113 @@
+---
+layout: blog
+title: "Introducing Kube-scheduler-simulator"
+date: 2024-07-xx
+slug: introducing-kube-scheduler-simulator
+canonicalUrl: https://kubernetes.dev/blog/2024/07/xx/introducing-kube-scheduler-simulator/
```

Comment:
What's the reason for including the `canonicalURL` here? Is this blog mirrored from the [k8s.dev](https://www.kubernetes.dev/blog/) site? If so, review comments should be added to the original content there.

Reply:
Sorry, actually I just copied those tags from another blog post. 

I saw Kwok's blog post, which I actually copied tags from, have two articles in two websites.
- https://www.kubernetes.dev/blog/2023/03/01/introducing-kwok/
- https://kubernetes.io/blog/2023/03/01/introducing-kwok/

Would you recommend creating a blog post in kubernetes.dev too? What's the general difference between those two websites?

Reply:
https://www.kubernetes.dev/blog/ is aimed only at people contributing to Kubernetes
https://www.kubernetes.io/blog/ is aimed at people using to Kubernetes

Who is this article aimed at? I think people might be using this even though they are not Kubernetes contributors - right?

Reply:
> Who is this article aimed at? I think people might be using this even though they are not Kubernetes contributors - right?

Yes, correct. So, here's the place for kubernetes.io blog, right?
[Did you suggest](https://github.com/kubernetes/website/pull/47104#pullrequestreview-2214335131) making the same post on .dev too? Or, is having it only here fine?

Reply:
This article is in the right repo. No need to change it.

We never mirror from the general website to the contributor website, so there's also no need to open a PR against the contributor site for this article.